app/src/main/java/org/wikipedia/feed/configure/ConfigureFragment.java

```diff
@@ -176,6 +177,10 @@ private void prepareContentTypeList() {
                 i.remove();
                 continue;
             }
+            if (!AccountUtil.isLoggedIn() && feedContentType == FeedContentType.SUGGESTED_EDITS) {
```

Comment:
@cooltey This should actually be a change to the constructor in FeedContentType.  
`   SUGGESTED_EDITS(9, R.string.suggested_edits_feed_card_title, R.string.feed_item_type_suggested_edits, false,AccountUtil.isLoggedIn()) {
`
It will automatically be taken care of.

Reply:
Thanks! Done!